### PR TITLE
Initial checkin for Horizon Intra-Company Sharing Quickstart

### PR DESCRIPTION
**Pull request by sfc-gh-vsrihari**
Initial checkin for this quickstart and summit lab hands-on-lab

### CODE REVIEW

File: site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md [hunk 1]
```diff
@@ -0,0 +1,1506 @@
+authors: Matthias Nicola, Vinay Srihari
+id: horizon_intra_org_sharing
+summary: demonstrates Horizon features for intra-company sharing of data and applications 
+<!--- Categories below should be hyphenated, i.e., Getting-Started. Do not leave blank. Visit site for available categories. -->
```
Comment (iamontheinet) [inline site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md]: @sfc-gh-vsrihari Pl delete this comment.

File: site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md [hunk 2]
```diff
@@ -0,0 +1,1506 @@
...
+categories: horizon-access,data-sharing,summit-hol
```
Comment (iamontheinet) [inline site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md]: @sfc-gh-vsrihari See https://quickstarts.snowflake.com/ for valid categories

File: site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md [hunk 3]
```diff
@@ -0,0 +1,1506 @@
...
+LIMIT 10;
+```
+
+Try the same steps on the <mark>Azure Consumer Account</mark>
+
+> How does the refresh frequency for cross-cloud fulfillment impact the application upgrade?
+> (documentation on cross-region application upgrades [here](https://docs.snowflake.com/en/developer-guide/native-apps/versioning#upgrade-an-installed-app-across-multiple-regions))
+
+<!-- ------------------------ -->
+## Conclusion
```
Comment (iamontheinet) [inline site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md]: @sfc-gh-vsrihari Make the layout and structure of this section similar to https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html#8

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md (hunk 1 above)
Comment (sfc-gh-vsrihari) [reply site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md]: Done.

File: site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md (hunk 2 above)
Comment (sfc-gh-vsrihari) [reply site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md]: Added Data-Sharing. Can we add these to the list of categories in Quickstart, please?

File: site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md (hunk 3 above)
Comment (sfc-gh-vsrihari) [reply site/sfguides/src/horizon_intra_org_sharing/horizon_intra_org_sharing.md]: Done.

**Pull request closed without merging**